src/components/plots/stackedAreaPlot.ts

```diff
@@ -3,6 +3,7 @@
 module Plottable {
 export module Plots {
   export class StackedArea<X> extends Area<X> {
+    private _stackedExtent = [0, 0];
```

Comment:
`[]` might be more appropriate, since it conveys "no extent", which is subtly different from `[0, 0]`.